codice/ddf
# DDF-1886 Added admin ui for adding/editing local registry node information

#### What does this PR do?
Adds admin ui for adding/editing local registry node information. 
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@djblue @andrewkfiedler @harrison-tarr 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@shaundmorris

#### How should this be tested?
Start up ddf and then start the DDF Registry app. Once the app is started you can open up the app and see a new 'Local Node Information' tab. You can play around with setting/adding/removing nodes/fields and saving them.
#### Any background context you want to provide?
This was pulled from the ddf-registry repo.
#### What are the relevant tickets?
DDF-1886
#### Screenshots (if appropriate)
Main table view
![screen shot 2016-06-06 at 4 08 54 pm](https://cloud.githubusercontent.com/assets/5248090/15840775/0fd42654-2c01-11e6-9587-a961fdc7a014.png)

Edit dialog
![screen shot 2016-06-06 at 4 01 43 pm](https://cloud.githubusercontent.com/assets/5248090/15840776/14d93446-2c01-11e6-88ba-53e6630d65f3.png)

Validation error
![screen shot 2016-06-07 at 4 11 57 pm](https://cloud.githubusercontent.com/assets/5248090/15882626/472dcb2e-2cf5-11e6-954a-f7f1ff5a1361.png)


#### Checklist:
